### ISSUE TITLE
grails 3.3.1  

### ISSUE THREAD

**Issue by ibmsoft** [springfox/springfox-grails-integration#21]
current ,can not use in grails 3.3.1, is there any solution?

**Comment by dilipkrish** [comment]
Yes, planning to work on it in the next couple weeks

**Comment by ibmsoft** [comment]
any progress?

**Comment by dilipkrish** [comment]
Yeah. I'm currently stuck at a point where Im no as familiar with grails. Im getting a message asking me to first initialize the gorm context before I can introspect domain classes. If you could provide insight that will be much appreciated.

**Comment by ibmsoft** [comment]
may be you can do something  in plugin
eg:
 void doWithApplicationContext() {
        // TODO Implement post initialization spring config (optional)
    }

**Comment by dilipkrish** [comment]
Not very familiar So I pushed a PR #22 with work in progress to solicit help. I'll look into it for sure.

**Comment by krishnact** [comment]
You need to start the plugin after GORM has been initialized. The solution is to set autostartup to false using this stanza in application.yaml:
```
springfox:
    documentation:
      auto-startup : false

```
In your Bootstrap.groovy, get a bean reference to the plugin and start it from init.

```
class BootStrap {
// springfox bean
DocumentationPluginsBootstrapper documentationPluginsBootstrapper;

    def init = { servletContext ->
        // Lets start springfox bean
        documentationPluginsBootstrapper.start()
        // Other initialization code
        //
        //
    }
    def destroy = {
    }
}
```

**Comment by dilipkrish** [comment]
There is a PR #23, I plan to merge in soon. Thanks for your suggestion!

**Comment by TabbyC** [comment]
Any update on getting that PR merged?

**Comment by hemantseth0210** [comment]
@dilipkrish Do you have plan to merge the PR#23 anytime soon?

**Comment by gabivis** [comment]
Same question, Any plans to merge PR #23?
